examples/JsonWithReferences/Program.cs

```diff
@@ -165,17 +145,25 @@ static async Task Main(string[] args)
                 AutoRegisterSchemas = false,
                 SubjectNameStrategy = SubjectNameStrategy.TopicRecord
             };
+            
+            var jsonSchemaGeneratorSettings = new JsonSchemaGeneratorSettings
+            {
+                SerializerSettings = new JsonSerializerSettings
+                {
+                    ContractResolver = new DefaultContractResolver
+                    {
+                        NamingStrategy = new CamelCaseNamingStrategy()
+                    }
+                }
+            };

```

Comment:
Is the user always supposed to provide this? Won't it make sense that we keep this as default `JsonSchemaGeneratorSettings` for both `JsonSerializer` and `JsonDeserializer` if the user isn't providing any `JsonSchemaGeneratorSettings`?

Reply:
It's not required to provide this, this allows you to set the default strategy and avoid to add attributes like
`[JsonProperty("favoriteColor")]` to the fields, but that is also possible. I'd not change JsonSerializer defaults, also it would be a breaking change (see PersonPoco in tests).
I can add a comment about this.